.eslintrc

```diff
@@ -14,10 +14,6 @@
         ],
         "no-restricted-syntax": [
             "error",
-            {
-                "selector": "ImportDeclaration[source.value='react'] :matches(ImportNamespaceSpecifier, ImportSpecifier)",
-                "message": "Please use import React from 'react' instead."
-            },
```

Comment:
Using `React` namespace is our codestyle. So `import * as React from 'react'` should be the only option.
To accomplish this, change the rule to:
```json
{
    "selector": "ImportDeclaration[source.value='react'] :matches(ImportDefaultSpecifier, ImportSpecifier)",
    "message": "Please use `import * as React from 'react'` instead."
}
```